sdk/ai/azure-ai-resources/config.json

```diff
@@ -0,0 +1 @@
+{"subscription_id": "f375b912-331c-4fc5-8e9f-2d7205e3e036", "resource_group": "jingyizhu-rag-rg", "project_name": "jingyizhu-ai-project-11-10"}
```

Comment:
I guess we don't need this file?

Reply:
You are right. This file was generated for testing purposes. I will remove it